reclass/values/__init__.py

```diff
@@ -1,5 +0,0 @@
-# -*- coding: utf-8
-from __future__ import absolute_import
```

Comment:
just keep these, it's in all .py files. Better to have one there to avoid someone miss another day later.

Reply:
In the future I'd rather kep __init__.py clean, especially given the tendency to put actual code there (which is an antipattern). It could be left here to avoid corresponding imports in other modules, but it could lead to interesting issues with tests. I will, however, revert this change now.